### PR TITLE
[EDU-2609] feat: cells node compatibility

### PR DESCRIPTION
**Pull request by gabriel-azion**
https://aziontech.atlassian.net/jira/software/c/projects/EDU/boards/136?assignee=635c54ab548f1fe6f0c6ac38&selectedIssue=EDU-2609

### CODE REVIEW

File: src/content/docs/en/pages/devtools/azion-edge-runtime/compatibility/cells-node.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        return new Response("ok");
+    });
+}
+
+addEventListener("fetch", (event) => {
+    event.respondWith(handleRequest(event.request));
+});
+```
+
+**For more compatibility information**:
```
Comment (hannah-sk) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/compatibility/cells-node.mdx:62]:
```suggestion
For additional compatibility information:
```

**Review by hannah-sk — APPROVED** [review]
all good!

**Review by robsongajunior — APPROVED** [review]
lgtm

File: src/content/docs/en/pages/devtools/azion-edge-runtime/compatibility/cells-node.mdx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+The compatibility is listed below:
+
+## Node built-in
+
+### [async_hooks](https://nodejs.org/api/async_hooks.html)
+
+**Supported**: 🟡 Partially supported
+
+**Comments**: Only [AsyncLocalStorage](https://nodejs.org/api/async_context.html#class-asynclocalstorage) and [AsyncResource](https://nodejs.org/api/async_hooks.html#class-asyncresource) are implemented.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/compatibility/cells-node.mdx:22]:
```suggestion
**Comments**: only [AsyncLocalStorage](https://nodejs.org/api/async_context.html#class-asynclocalstorage) and [AsyncResource](https://nodejs.org/api/async_hooks.html#class-asyncresource) are implemented.
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**